packages/os/mark-successful-boot.service

```diff
@@ -8,6 +8,7 @@ After=multi-user.target
 Type=oneshot
 RemainAfterExit=true
 ExecStart=/bin/signpost mark-successful-boot
+ExecStartPost=/usr/bin/metricdog send-boot-success
```

Comment:
I think we may want the `-` prefix so that service startup continues if we can't send a metric ping. (https://man7.org/linux/man-pages/man5/systemd.service.5.html)

Reply:
Added in [b8b5c3a](https://github.com/bottlerocket-os/bottlerocket/compare/153acff977082318fef8fc8e61024b26574c362e..b8b5c3ad92298db2c46262eb2b6c5b6ea3f60187)